packages/server/src/api/util/catch-async-route.ts

```diff
@@ -0,0 +1,19 @@
+import { NextFunction, Request, Response, RequestHandler } from 'express';
```

Comment:
similar to wrap-request, but:
- don't add data prop to response
- fwd error to error handler middleware

Reply:
Should we remove `wrap-request`?

Reply:
didn't want to break the v2 apis